Makefile

```diff
@@ -15,7 +15,7 @@ HELM_OPTS ?= --set serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn=${K
       		--set clusterName=${CLUSTER_NAME} \
 			--set clusterEndpoint=${CLUSTER_ENDPOINT} \
 			--set aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME}
-TEST_FILTER ?= .*
+TEST_FILTER ?= TestIntegration
```

Comment:
this is used in the regular tests as well in the `test` and `battletest` target

Reply:
wow good catch